showkase-screenshot-testing-paparazzi-sample/src/main/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/Scrollable.kt

```diff
@@ -0,0 +1,46 @@
+package com.airbnb.android.showkase.screenshot.testing.paparazzi.sample
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.airbnb.android.showkase.annotation.ShowkaseComposable
+
+@Composable
+fun VerticalScrollSample() {
+    Column(
```

Comment:
Is there a reason why we are not using LazyColumn here? 😄

Reply:
no, was just added as a sample. No particular reason to add it or skip it 😅